backend/src/main/java/gov/cdc/usds/simplereport/service/email/EmailProviderTemplate.java

```diff
@@ -9,5 +9,8 @@
   ACCOUNT_REQUEST,
 
   // identity verification failed email (to account requester)
-  ID_VERIFICATION_FAILED;
+  ID_VERIFICATION_FAILED,
+
+  // message to account requester who abandoned without successful id verification
```

Comment:
very minor nit: I think we should change this to something like "requester who didn't complete automatic id verification", because as I understand this won't just go to the abandoners but also to folks that tried to go through Experian and failed.